content/guide/install-uninstall.md

```diff
@@ -323,4 +323,80 @@ To uninstall Quire:
 
 ## Troubleshooting
 
-TK
+**Errors installing Prince**
+
+When installing versions of Prince newer than 12.5 you will need to follow these steps:
+
+1. Copy and paste the following code:
+
+    ```text
+    cd Downloads/prince-12.5-macosx
+    sudo ./install.sh
+    ```
+
+The key line is **prince-XX.X-macos(x)**
+
+2. Go to the Downloads folder to make sure the Prince zip file is there.
+
+3. Make sure XX.X matches the current version of Prince that you are using.
+
+4. The line may end in macos or macosx. Make sure the it matches exactly what the program looks like in the download folder.
+
+
+**Downloading the Quire CLI to your computer from GitHub through the Terminal**
+
+If you have two-factor authentication set-up, you may need to create a personal access token in GitHub to get Quire CLI to download properly.
+
+Follow these steps:
+
+1. Run
+
+    ```text
+    git clone https://github.com/gettypubs/quire-cli.git
+    ```
+
+2. For username: enter your Github Username
+
+   For Password: **follow the directions below.**
+
+3. In the upper-right corner of any page, click your profile photo, then click Settings.
+
+4. In the left sidebar, click Developer settings.
+
+5. In the left sidebar, click Personal access tokens.
```

Comment:
I always hesitate a little about describing interface details so explicitly. As you saw with GitHub Desktop and that first tutorial you did, when the interface changes, directions like this would have to change as well, and I don't think we have a good system for tracking and checking that kind of thing.

Luckily in this instance, there's also a URL that goes straight to tokens and will work for users universally: https://github.com/settings/tokens